### PR TITLE
NOTION - huge update - 2

### PR DESCRIPTION
**Pull request by Mirm0n**
I made a lot of fixes to the notion website. I had previously made a pull request but I made a lot of changes so I decided to create a new one instead for approval
190619918-34d31f98-1645-49be-abe9-2c0be809b6c8

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -14099,14 +14099,14 @@ CSS
...
-.api-section-image .next-image
-.visual .next-image
+.next-image
 .logo svg > :first-child
+.notranslate::after
+.logo-limit-height
+.jsx-2604928324.mobile-actions
+.jsx-fb3129685d85c2fa.menu-toggle
+.jsx-422720195.logo
+.jsx-1042831150.logo
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:14109]: The `.jsx-xxxxx` ones should be removed, they are randomized and generally doesn't last long before they are replaced by the site.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -14115,6 +14115,21 @@ body {
 .notion-divider-block div div {
     border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;
 }
+.header.jsx-2604928324 {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.person.jsx-2691780686{
+    background-color: var(--darkreader-neutral-background) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:14123]: These selectors can be merged with `body` at line 14112

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -14115,6 +14115,21 @@ body {
...
+.notranslate { 
+    color: var(--darkreader-neutral-text) !important;
+}
+.notion-selectable notion-callout-block{
+background-color: var(--darkreader-neutral-background) !important;
+}
+.jsx-422720195.header{
+background-color: var(--darkreader-neutral-background) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:14132]: These selectors can be merged with `body` at line 14112

**Comment by pl0xy** [comment]
Or you could just make those requested changes and merge? obviously without the first set, cause they do randomise.

**Comment by Gusted** [comment]
> Or you could just make those requested changes and merge? obviously without the first set, cause they do randomise.

I can only test partially of the changes, otherwise I would have already done it. Closing.

**Pull request closed without merging**